core/server/data/migrations/versions/4.3/01-add-products-table.js

```diff
@@ -0,0 +1,9 @@
+const addTable = require('../../utils').addTable;
```

Comment:
Not even worth mentioning but is there a reason you don't use object destructuring here? 😝

Reply:
I couldn't remember what the function was called so I did `require('../../utils').` and let autocomplete show me the options :joy:

Reply:
Hahahaha 🤣

Reply:
Switched this out for destructuring when I added ze comment FYI!